src/grid/rtransform.py

```diff
@@ -52,8 +52,8 @@ def deriv2(self, x):
     def deriv3(self, x):
         """Abstract method for 3nd derivative of transformation."""
 
-    def generate_radial(self, oned_grid):
-        """Generate a radial grid by transforming the OneDGrid.
+    def generate_grid(self, oned_grid):
```

Comment:
Maybe call it "trans1d_grid" or something to make it clear that the grid here is the transformation of the oned_grid you read in? Of course it is (also) a 1d grid....

Reply:
or call it generate_tfgrid or generate_tf1dgrid to harmonize with other nomenclature?